src/argv.ts

```diff
@@ -46,8 +46,17 @@ export class Argv {
         argv.injectDotenv(`${argv.cwd}/.gitlab-ci-local-env`, args);
 
         if (!argv.shellExecutorNoImage && argv.shellIsolation) {
-            writeStreams?.stderr(chalk`{black.bgYellowBright  WARN } --shell-isolation does not work with --no-shell-executor-no-image\n`);
+            writeStreams?.stderr(chalk`{black.bgYellowBright  WARN } --shell-isolation does not work with --shell-executor-no-image\n`);
```

Comment:
not really a typo, 

https://yargs.js.org/docs/#api-reference-booleankey
`--no-shell-executor-no-image` is an alias for `--shell-executor-no-image=false`